### ISSUE TITLE
Show errors more clearly for prisma seed

### ISSUE THREAD

**Issue by zwhitchcox** [prisma/prisma#4322]
**Is your feature request related to a problem? Please describe.**
errors in `prisma seed` are shown as html in console

**Describe the solution you'd like**
show errors more clearly

**Additional context**

This is the error I get

```
PS C:\Users\Zane\dev\react-apollo\server> prisma seed

ERROR: GraphQL Error (Code: 400)

{
  "error": "<!DOCTYPE html>\n<html>\n  <head>\n    <meta charset=\"utf-8\">\n    <title>Bad Request – 400</title>\n    <style>\n      html, body {\n        margin: 0;\n        width: 100%;\n        height: 100%;\n
       display: flex;\n        justify-content: center;\n        align-items: center;\n      }\n\n      body {\n        font-family: -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Helvetica, Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\";\n        font-size: 15px;\n      }\n\n      .Error {\n        font-size: 1.35em;\n        \n          color: #6061BE;\n        \n      }\n\n
  .Error .message {\n        font-weight: 200;\n        letter-spacing: 0.095em;\n      }\n\n      .Error .message a {\n        text-decoration: none;\n        color: inherit;\n        \n          border-bottom: 1px dotted #6061BE;\n        \n      }\n\n      .Error .status {\n        font-weight: 700;\n      }\n\n
 .Error .code {\n        display: none;\n      }\n\n      .Error .dot {\n        font-weight: 100;\n      }\n\n      @media screen and (max-width: 800px) {\n        body {\n          font-size: 10px;\n        }\n\n        .Error {\n          display: flex;\n          flex-direction: column;\n        }\n\n        .Error
.status {\n          font-size: 1.1em;\n        }\n\n        .Error .dot {\n          display: none\n
  }\n      }\n    </style>\n  </head>\n  <body>\n    <div class=\"Error\">\n      <span class=\"status\">Bad Request</span>\n      <span class=\"code\">400</span>\n      <span class=\"dot\">•</span>\n      \n
  <span class=\"message\">Please try your request again or contact support.</span>\n      \n    </div>\n  </body>\n</html>",
  "status": 400
```

**Comment by zwhitchcox** [comment]
Also, side not, this error comes from a breaking change in the new release. Not sure what it is, but when I rolled it back to 1.29.1, it fixed the error

**Comment by zwhitchcox** [comment]
Actually, rolling back also fixed the error output...so maybe this should just be a bug?

**Comment by pantharshit00** [comment]
We have fixed this bug. The fix is now available in the latest beta `1.31.0-beta.2`

**Comment by divyenduz** [comment]
This is available in the latest master now, please install it via `npm install -g prisma`
